src/components/asset/Asset.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import get from 'lodash.get';
+
+const Asset = ({ data }) => {
+    const { url, description, contentType } = get(data, 'target.fields.file', {});
+
+    if (contentType.includes('image')) {
+        return <img data-type="image" src={ url } alt={ description } />;
```

Comment:
This is more of a discussion/question, but: Shouldn't we allow the user to override the components that render each `nodeType`?

I commented here because I think assets are the best showcase for this. In the near future we will have `@moxy/react-contentful-image`. Imagine the user wants to make use of the Contentful Image API in the middle of a RichText block. It would be a good idea to let the user provide a custom `Asset` that renders an image using  `@moxy/react-contentful-image`.

I realize this would make the implementation more complicated, especially taking into account nodeType components that render another level of the `Content` component, but I think we should consider this.

Thoughts?

Reply:
One thing I want to do in the future is to use `@moxy/react-contentful-image` as the default image rendering component for this package.

I also have that in mind, which is to delegate the render of `nodeTypes` to other components. Just like we do with entries and with `entryMap`. 

Although, as you said, I think that would increase the complexity of the implementation right now.

Should I look into it right now, or should we open an issue to look into that later on?
Also, should every `nodeType` have the possibility to "override" the default render method?

Reply:
I think is more important to have a functioning version first, and consider this as a future improvement. We should discuss which `nodeType`s should be overridable and how we should approach it.

All in all, I just wanted to start the discussion, the current way will probably be good enough for projects in the near future, and if it isn't we can tackle it when we run into it.

I can open the issue after this PR is merged, if you want

Reply:
Opened an issue!

I will resolve this discussion and we can further discuss it there: https://github.com/moxystudio/react-contentful-rich-text/issues/2